tools/tests/apps/dynamic-import/.meteor/release

```diff
@@ -1 +1 @@
-METEOR@1.6.1
+METEOR@1.7.0.3
```

Comment:
If I recall, this (as in the `dynamic-import` test app's Meteor version) is intentionally held back until the version is officially released, correct?

That is to say, this is intentionally not 1.7.0.4 yet.

Reply:
That's right. I will update it again once the final release commit is merged into `devel`.